### ISSUE TITLE
IMPORTANT: Rewrote history of project

### ISSUE THREAD

**Issue by alexandru** [alexandru/sincron#4]
@aoprisan I've rewritten the history of the project to hide my emails behind noreply@alexn.org. This is because I'm having a GitHub-related spam problem. I'm following the tutorial here: https://help.github.com/articles/changing-author-info/

You have to rebase, like so:
1. DO NOT do a `git pull`
2. `git fetch`
3. for each local branch forked from master: `git rebase --onto origin/master master <local_branch>` 
4. for each remote branch: `git rebase --onto origin/<remote-branch> <remote-branch> <local_branch>` 
   - in other words, you probably have to do: `git rebase --onto origin/aref-macro aref-macro aref-macro`
5. `git pull --force`

Sorry for the inconvenience.

**Comment by aoprisan** [comment]
Done, thanks.
